unit_tests/test_tester.py

```diff
@@ -196,6 +199,30 @@ def _validate_results(self):
         assert self.final_event.test_status == 'FAILED'
 
 
+class CriticalErrorNotCaught(ClusterTesterForTests):
```

Comment:
Class name doesn't have "Test" word. Will be picked up by the test runner at all?

Reply:
It is strange, but it is picking it up, but I have added it anyway